client/js/components/assessmenttable/DpNewStatement.vue

```diff
@@ -170,7 +170,7 @@ export default {
     ]),
 
     phases () {
-      if (this.institutionSelected && hasPermission('field_show_internal_procedure_phases_in_dropdown')) {
```

Comment:
Why can this permission be removed?

Reply:
Because there is an existing permission doing the same: `field_statement_phase`

Reply:
but you did not replace the permission but entirely remove it!

Reply:
long story short (as i understood it from @sierrazamudiodemos and looking through historic git repos):
- `field_show_internal_procedure_phases_in_dropdown` was introduced to replace a twig override as a refactoring
- this permission however was not properly set in all projects, resulting in it being not effective 
- instead of choosing client-side which phases to display, that logic is now implemented in the backend, so that `availableProcedurePhases` always holds the correct set of phases (internal or external)